app/lib/permission_checker.rb

```diff
@@ -1,7 +1,8 @@
 class PermissionChecker
   GDS_EDITOR_PERMISSION = "gds_editor".freeze
   EDITOR_PERMISSION = "editor".freeze
-
+  PREVIEW_DESIGN_SYSTEM = "Preview design system".freeze
+  PREVIEW_NEXT_RELEASE = "Preview next release".freeze
```

Comment:
Could you explain in the commit message that adds these what you intend to use them for? I have a vague understanding from working on Whitehall and without that wouldn't understand at all - so I imagine anyone other than me wouldn't understand. Even with my understanding I don't know how you intend to approach transitioning Manuals so would be good to have documented at what points we'd use either of these?

(I also somewhat imagine you'll only need one permission for Manuals? is the design system transition going to be shipped in multiple releases?)